index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>porfolio project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+ <header class="header-section">
+<a class="logo">My Logo</a>
+<nav> <img class=hamburger src="images/Union.png" alt=""></nav>    
+<!-- <a class="porfolio"href="#">Porfolio</a>
+<a href="#about" class="about">About</a>
+<a href="#contact" class="contact">Contact</a> -->
+</header>
+<section class="main-section">
+    <h1>I'm Therese TUYISABE<br> Glad to see you</h1>
+    <div class="primary text">
+<p class="text1">
+I am software developer!I can help you build a product feature or 
+website lookthrough sameof my work and experience!
+if you like what you see and have a project you need coded don't hestiate to contact me.
+</p>
+<a class="text2">Let's Connect</a></div>
+<div class="social-icon">
+<a href="https://twitter.com" class="twitter"> <img src="images/twitter.png" alt=""></a>
+<a href="https://linkedin.com" class="linkedin"><img src="images/Linkedin icon.png" alt=""></a>
+<a href="https://medium.com" class="medium"><img src="images/medium (2).png" alt=""></a>
+<a href="https://github.com" class="github"><img src="images/github.png" alt=""></a>
+<a href="https://angellist.com" class="angellist"><img src="images/angellist.png" alt=""></a>
```

Comment:
As the previous reviewer mentioned, social links are not on the page. Please fix them.
<img width="511" alt="Screenshot 2023-01-19 at 8 10 24 PM" src="https://user-images.githubusercontent.com/25887253/213513364-fdd926f5-02c7-4753-a430-06c0c22a58fa.png">
<img width="511" alt="Screenshot 2023-01-19 at 8 10 35 PM" src="https://user-images.githubusercontent.com/25887253/213513371-df8b75f2-5478-49a9-9439-9833419318f2.png">